### DISCUSSION TITLE
TF 2.3 Meeting - 26th Oct

### DISCUSSION THREAD

**Discussion by stadmill** [OSIPI/DCE-DSC-MRI_CodeCollection#115] [General] [answered]
Hi everyone, I have scheduled a call for the 26th at 9pm AEDT / 10am UTC / 6am EDT . Join on Teams - 

Meeting ID: 494 622 142 729 
Passcode: hirtYp

**Comment by stadmill** [comment, marked as answer]
Moved to Thursday 3 Nov, same time.

**Reply by sudarshanragu** [reply]
@stadmill Is it possible to access the meeting minutes anywhere ? I was unable to make it today on account of being sick, but would like to catch up on any updates from today's meeting. Thanks!

**Reply by LucyKershaw** [reply]
It will be just as soon as I've written them!

**Reply by LucyKershaw** [reply]
Also, get well soon

**Reply by sudarshanragu** [reply]
Thank you @LucyKershaw !

**Reply by LucyKershaw** [reply]
[Minutes_Oct.pdf](https://github.com/OSIPI/DCE-DSC-MRI_CodeCollection/files/9940071/Minutes_Oct.pdf)

Here are the meeting minutes with actions included
